src/js/components/molecules/wizard-step-download-list/wizard-step-download-list.ts

```diff
@@ -28,6 +29,13 @@ export class WizardStepDownloadList extends Vue implements WizardStepDownloadLis
         this.selected = index;
     }
 
-    handleSubmit(e) {
+    handleSubmit() {
+        window.location.assign(this.file);
+
+        // @ts-ignore
+        if (window.$u) {
+            // @ts-ignore
+            window.$u.store.setSetting('robot', true);
```

Comment:
Could you add a comment to explain what this does?

Reply:
Will explain later ;-)